src/strategies/pokt-network-pda/index.ts

```diff
@@ -0,0 +1,231 @@
+import fetch from 'cross-fetch';
+import { subgraphRequest } from '../../utils';
+import { EnumType } from 'json-to-graphql-query';
+
+// Interfaces
+interface PoktNetworkOptions {
+  arweave_network: 'MAINNET' | 'DEVNET';
+  owner_address: string;
+  decimals: number;
+}
+
+interface ArweaveBlockResponse {
+  transactions: {
+    edges: [
+      {
+        node: {
+          id: string;
+        };
+      }
+    ];
+  };
+}
+
+interface StoreDomainBlock {
+  point: number;
+  PDAs: string;
+}
+
+interface StakerDomainBlock<Block> {
+  validator?: Block;
+  gateway?: Block;
+  'liquidity provider'?: Block;
+}
+
+interface PDAScores {
+  [walletAddress: string]: {
+    citizen?: StoreDomainBlock;
+    builder?: StoreDomainBlock;
+    staker?: StakerDomainBlock<StoreDomainBlock>;
+  };
+}
+
+interface SumHousesPoints {
+  builder_points: number;
+  validator_and_liquidity_provider_points: number;
+  gateway_points: number;
+}
+
+// Major constants
+export const author = 'pokt-network';
```

Comment:
@ahmadyazdanii Should be coming from `pokt-network`
s github or can change it your name

Reply:
```suggestion
export const author = 'ahmadyazdanii';
```

Reply:
Can I use Microflow-xyz instead of my username?

Reply:
Yep for sure